src/System.CommandLine.Rendering/CursorControlSpan.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.CommandLine.Rendering
+{
+    [DebuggerDisplay("{" + nameof(Name) + "}")]
+    public class CursorControlSpan : Span
+    {
+        public CursorControlSpan(string name, int row = 0, int column = 0)
+        {
+            Name = name;
+            Row = row;
+            Column = column;
+        }
+        public override int ContentLength => 0;
+
+        public string Name { get; }
+        public int Row { get; }
+        public int Column { get; }
```

Comment:
Are `Row` and `Column` being used? They don't look applicable  to all of the `CursorControlSpan` variants.